README.md

```diff
@@ -1,133 +1,96 @@
   * [Best Practices Badge](#best-practices-badge)
-  * [Non-Updated Items](#non-updated-items)
   * [License](#license)
   * [Formats](#formats)
   * [Installation](#installation)
   * [Vulnerability reporting](#vulnerability-reporting)
   * [Adjusting the Landscape View](#adjusting-the-landscape-view)
 
-The [CNCF](https://www.cncf.io) Cloud Native Landscape Project is intended as a map through the previously uncharted terrain of cloud native technologies. This attempts to categorize most of the projects and product offerings in the cloud native space. There are many routes to deploying a cloud native application, with CNCF Projects representing a particularly well-traveled path. It has been built in collaboration with [Redpoint Ventures](https://www.redpoint.com/) and [Amplify Partners](http://www.amplifypartners.com/).
+The [Interactive Mobility Data Landscape](https://nrel.github.io/mobility_landscape/) is a project intended as a map through the previously uncharted, growing mobility data science field. This attempts to categorize most of the data sources, specifications, and tools in the field of mobility. It has been built by the National Renewable Energy Laboratory's (NREL) Mobility, Behavior, and Advanced Powertrain (MBAP) research group as an internship project.
 
-The software for the interactive landscape has been extracted to https://github.com/cncf/landscapeapp where it is used for other landscapes as well. This repo includes all of the data and images specific to the CNCF landscapes.
-
-## Trail Map
-
-The Cloud Native Trail Map provides an overview for enterprises starting their cloud native journey.
-
-[![CNCF Trail Map](https://github.com/cncf/trailmap/blob/master/CNCF_TrailMap_latest.png)](https://raw.githubusercontent.com/cncf/trailmap/master/CNCF_TrailMap_latest.png)
+The software for the interactive landscape has been extracted to https://github.com/NREL/mobility_landscapeapp/tree/master/src. This repo includes all of the data and images specific to the mobility data landscape.
 
 ## Interactive Version
 
-Please see [landscape.cncf.io](https://landscape.cncf.io).
+Please see [nrel.github.io](https://nrel.github.io/mobility_landscape/).
 
 ## New Entries
 
-* [Cloud native](https://github.com/cncf/toc/blob/master/DEFINITION.md) projects with at least 300 GitHub stars that clearly fit in an existing category are generally included. Put the project in the single category where it best fits.
-* We generally will only list a company's product in one box, to represent its major or best-known offering. We occasionally make exceptions for large companies. Note that if we allowed listing the same product or project in multiple boxes, the over 600 logos on the landscape would multiply to many times that many.
+These are the steps to go through to upload a new entry:
+
+1. After opening a pull request to landscape.yml, add a new entry within this file. Follow the order and format of previous entries.
```

Comment:
pull request is typically the last step. You first add an entry, and then commit it and then generate a pull request